src/random.h

```diff
@@ -236,4 +236,8 @@ bool Random_SanityCheck();
  */
 void RandomInit();
 
+/**
+ * Flag to make GetRand return the same number
+ */
+extern bool g_mock_deterministic_tests;
```

Comment:
nit: Declare `extern` in test code so it's not otherwise accessible in the non-test code?

Reply:
Done